### PR TITLE
Rename file on new verision upload

### PR DESCRIPTION
**Pull request by tmpacifitech**
Part of https://github.com/SeedCompany/cord-field/issues/786

### CODE REVIEW

**Review by CarsonF — CHANGES_REQUESTED** [review]
This is no longer needed
https://github.com/SeedCompany/cord-api-v3/blob/develop/src/components/file/file.service.ts#L401-L405

**Comment by tmpacifitech** [comment]
> This is no longer needed https://github.com/SeedCompany/cord-api-v3/blob/develop/src/components/file/file.service.ts#L401-L405

Fixed.

File: src/components/file/file.service.ts [hunk 1]
```diff
@@ -264,6 +264,12 @@ export class FileService {
       await this.bucket.moveObject(`temp/${uploadId}`, uploadId);
     }
 
+    // Change the file's name to match the latest version name
+    // Since defined files are explicitly referenced by a named property
+    // a consistent name is not required and automatically updating it is more
+    // convenient for consumption.
```
Comment (CarsonF) [inline src/components/file/file.service.ts:270]: This comment doesn't make sense here
```suggestion
```

**Review by CarsonF — CHANGES_REQUESTED** [review]

**Review by CarsonF — APPROVED** [review]
🤘

**Pull request merged**